src/main/java/com/google/cloud/genomics/utils/grpc/GenomicsChannel.java

```diff
@@ -26,10 +36,13 @@
 
 import javax.net.ssl.SSLException;
 
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.genomics.utils.GenomicsFactory;
+
 /**
  * A convenience class for creating gRPC channels to the Google Genomics API.
  */
-public class GenomicsChannel extends Channel {
+public class GenomicsChannel extends ManagedChannel {
```

Comment:
Good question Eric.  I made this class when we needed to expose shutdown for the hangs https://github.com/googlegenomics/utils-java/commit/042a40622aea0094b98202fed865364e4836a335  ManagedChannel did not exist at that time, only ChanneImpl.

I'll refactor now - thanks.